bin/gftools-add-axis.py

```diff
@@ -102,6 +102,10 @@ def main(font: str):
     axis_proto.max_value = fvar_axis.maxValue
     # Input values for this axis must aligned to 10^precision
     axis_proto.precision = 1  # ask user?
+    # Some parameter I don't know what it does
+    axis_proto.fallback_only = false
```

Comment:
Would it always be false? I understand not.

Reply:
I don’t know, I wanted to ask more details about it to you at the meeting, and the eng team if needed. I just see that all textprotos are having that parameter now with false as value so I thought it could be the default in this script?

Reply:
Evan said: The fallback_only was added to communicate to the front end how to present the axis in the UI. If set to true, the UI would present it as a dropdown (or radio buttons) instead of a slider. Cursive is the only axis currently using it. 

Eventually, the slider option could be set as default in the API, and the field would only be added when in need to override it (similar to the default value). But until the API is updated, it seems to leave it predefined as "false" here would be ok.